record-and-playback/presentation/scripts/publish/presentation.rb

```
@@ -39,7 +39,7 @@
 filepathPresOverride = "/etc/bigbluebutton/recording/presentation.yml"
 hasOverride = File.file?(filepathPresOverride)
 if (hasOverride)
-  presOverrideProps = YAML::load(File.open(filepathPresOverride))
+  presOverrideProps = YAML::load(File.read(filepathPresOverride))
```

RuboCop (1.86.2)

[convention] Do not use `::` for method calls.

(Style/ColonMethodCall)